cmd/mesh/manifest.go

```diff
@@ -142,12 +142,22 @@ func genApplyManifests(setOverlay []string, inFilename string, force bool, dryRu
 		return fmt.Errorf("failed to apply manifest with kubectl client: %v", err)
 	}
 	gotError := false
+	disabledComponents := []string{}
+	for cn := range manifests {
+		if isDisabledComponent(out[cn]) {
+			disabledComponents = append(disabledComponents, string(cn))
+		}
+	}
+	l.logAndPrint("\nComponents not enabled: ", strings.Join(disabledComponents, ", "))
+
 	for cn := range manifests {
 		if out[cn].Err != nil {
-			cs := fmt.Sprintf("Component %s install returned the following errors:", cn)
+			cs := fmt.Sprintf("Component %s returned the following errors:", cn)
```

Comment:
I suggest leaving as before. It's the install of the component that returned errors, not the component itself.

Reply:
The problem here is we also support 'uninstall' (or prune) of a disabled component.
Maybe the right thing to do here is to differentiate the actions here and give messages accordingly.

Reply:
how about changing install to "update"?